### PR TITLE
new long ponytail thing that works with hats

### PR DESCRIPTION
**Pull request by Sha-Seng**
- add: Added Gathered Low hairstyle for humans and every species that uses their hair

### CODE REVIEW

**Comment by formlessnameless** [comment]
please move this into the impstation file structure, then I'll merge it

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml [hunk 1]
```diff
@@ -971,6 +971,13 @@
   sprites:
     - sprite: Mobs/Customization/human_hair.rsi
       state: fringetail
+- type: marking
+  id: HumanHairGatheredlow
+  bodyPart: Hair
+  markingCategory: Hair
+  sprites:
+    - sprite: Mobs/Customization/human_hair.rsi
```
Comment (hivehum) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:979]: change this to _impstation/mobs/customization/human_hair.rsi and it wont error when you migrate the rest of the file structure

Comment (hivehum) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:979]: feel free to use pr #548 for reference on how to add cosmetics

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml [hunk 2]
```diff
@@ -1377,11 +1377,17 @@
...
       state: longbow
-
 - type: marking
   id: HumanHairFrenchBraid
   bodyPart: Hair
   markingCategory: Hair
   sprites:
     - sprite: Mobs/Customization/human_hair.rsi
       state: frenchbraid
+- type: marking
```
Comment (hivehum) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:1387]: this edit needs to be moved into its own file, as does the locale file edit. we should not be altering upstream files at all anymore

**Review by formlessnameless — APPROVED** [review]

**Pull request merged**